import javax.naming.event.NamingExceptionEvent;

public class logica {
    public static void main(String[]args){
    int a, b;
    double resultado;
    a=10;
    b=5;
    resultado = a/b;
    System.out.println(resultado);

    
}


}
